docs/HyperIndex/Guides/configuration-file.mdx

```diff
@@ -59,6 +51,7 @@ Should there be multiple contract addresses from which events should be indexed
 
 In the configuration you can optionally pass the file path to the abi for a contract in the `abi_file_path` field or you can specify the event signature in the event field.
 
+After you have set up your config file and the scheme, you are ready to generate the indexing code required to write the event handlers.
```

Comment:
This lines looks a little bit off here

Reply:
I've actually run the doc through chat gpt for editing and I think it looks better now.